docker/caffe2/jenkins/common/install_rocm.sh

```diff
@@ -2,8 +2,18 @@
 
 set -ex
 
+# TODO: This script should install a SPECIFIC ROCM_VERSION, but actually
+# it ignores all values of ROCM_VERSION which are not nightly.  Ugh!
 [ -n "$ROCM_VERSION" ]
 
+install_hip_nightly() {
```

Comment:
what does this install? will it override stuffs installed in install_ubuntu?

Reply:
Yes, it builds and then installs some new deb files.